lib/orangetheses/fetcher.rb

```diff
@@ -33,7 +35,7 @@ def self.config_yaml
     end
 
     def self.env
-      ENV['ORANGETHESES_ENV'] || 'development'
+      ENV['RAILS_ENV'] || 'development'
```

Comment:
@jrgriffiniii so that I can test the rake task I will run it in bibdata-staging setting RAILS_ENV=production so that it will point to https://dataspace.princeton.edu/rest and not to https://dataspace-staging.princeton.edu/ (staging has an expired certificate this is why it's failing to query the theses.)